internal/pkg/cli/validate.go

```diff
@@ -212,7 +212,7 @@ func validateJobType(val interface{}) error {
 	if !ok {
 		return errValueNotAString
 	}
-	return validateWorkloadType(jobType, manifest.JobTypes, "job")
+	return validateWorkloadType(jobType, manifest.JobTypes(), "job")
```

Comment:
maybe make "job" a constant as well for consistency?

Reply:
Done!